### ISSUE TITLE
The name 'weakly quasi-coherent'

### ISSUE THREAD

**Issue by felixwellen** [felixwellen/synthetic-zariski#35]
We call a module bundle on a type weakly quasi-coherent (wqc), if its sections on opens are given by (algebraic) localization (in some sense). At some occasions we concluded that the name is not so great, since these module bundles are not really an analogue of quasi-coherent sheaves. I don't remember the details of these past discussions, but I would argue that in general, our theory (in particular wqc modules and their cohomology, pullbacks and push-forwards) diverts from the classic story, so I think it would be good to also divert with our names a bit more. 
I think just calling the bundles "local module bundles" would be a good fit. Happy to hear opinions on that.

**Comment by iblech** [comment]
Right: Let $S$ be a base scheme. A wqc module bundle in the big Zariski topos of $S$ is quasicoherent in the sense that its restrictions to all $\mathrm{Sh}(X)$, where $X$ ranges over all (l.o.f.p.) $S$-schemes, is quasicoherent. But it need not be quasicoherent in the sense that it originates from a quasicoherent module on $S$ itself. A wqc module can still be "chimeratic" / "shape-shifting". A particularly drastic example is provided by kernels.
